rabbithole_test.go

```diff
@@ -103,6 +103,9 @@ var _ = Describe("Rabbithole", func() {
 
 			Ω(res.Node).ShouldNot(BeNil())
 			Ω(res.StatisticsDBNode).ShouldNot(BeNil())
+			Ω(res.MessageStats).ShouldNot(BeNil())
+			Ω(res.MessageStats.DeliverGetDetails).ShouldNot(BeNil())
+			Ω(res.MessageStats.DeliverGetDetails.Rate).Should(BeNumerically("==", 0))
```

Comment:
`>=` is a wiser choice: what if the rate happens to be > 0 for a moment, e.g. because the test suite changes over time?